bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ClientConfiguration.java

```diff
@@ -610,7 +610,7 @@ public int getExplictLacInterval() {
     /**
      * Set the interval to check the need for sending an explicit LAC.
      * @param interval
-     *        Number of seconds between checking the need for sending an explict LAC.
+     *        Number of milli seconds between checking the need for sending an explict LAC.
      * @return Client configuration.
      */
     public ClientConfiguration setExplictLacInterval(int interval) {
```

Comment:
This feels really bad from a backwards comparability standpoint.  Could we add in a new method that lets you set milliseconds but update this API to call into the other but converting seconds to milliseconds?